next/components/common/Figure/figure.styles.ts

```diff
@@ -1,16 +1,16 @@
 import { textBody400 } from "@clevercanary/data-explorer-ui/lib/styles/common/mixins/fonts";
 import styled from "@emotion/styled";
 
-export const ImageWithCaption = styled.div`
+export const Figure = styled.figure`
```

Comment:
Refactored the image with caption component - I need a standalone `Image` component...